pkg/cloudprovider/providers/vsphere/vsphere_util.go

```diff
@@ -18,15 +18,30 @@ package vsphere
 
 import (
 	"context"
+	"github.com/vmware/govmomi"
 	"os"
 	"runtime"
 	"strings"
 )
 
 // Reads vSphere configuration from system environment and construct vSphere object
 func GetVSphere() (*VSphere, error) {
+	cfg := getVSphereConfig()
+	client, err := GetgovmomiClient(cfg)
+	if err != nil {
+		return nil, err
+	}
+	vs := VSphere{
```

Comment:
why not `vs := &VSphere{ ... ` ?
Then remove `&` below? There may be a good reason for not doing this but it is a fairly common pattern.